docs/manuals/core/config/data-node-config.md

```diff
@@ -305,7 +305,7 @@ following parameters can be provided:
 - _**db_driver**_ represents the database driver that will be used by Taipy.<br/>
     The default value of _db_driver_ is "ODBC Driver 17 for SQL Server".
 - _**exposed_type**_ indicates the data type returned when reading the data node (more
-  examples of reading from SQL Table data node with different _exposed_type_ is available on
+  examples of reading from SQL table data node with different _exposed_type_ is available on
```

Comment:
"table data node" -> "table data nodes", I guess
"is available on" -> "are available in the"